explainer.md

```diff
@@ -17,86 +15,140 @@ The two drafts apply to both TLS server certificates, where the server authentic
 
 ## Introduction
 
-[TLS](https://www.rfc-editor.org/rfc/rfc8446) uses [X.509 certificates](https://www.rfc-editor.org/rfc/rfc5280) to associate the authenticating party's, or *subscriber's*, TLS key with its application identifiers, such as DNS names. These associations are signed by some certificate authority (CA). The peer, or *relying party*, curates a set of CAs that are trusted to only sign correct associations, which allows it to rely on the TLS to authenticate application identifiers. Typically the subscriber is the server and the relying party is the client.
+While they differ in approach, both drafts aim to solve the same problem: **enable [TLS](https://www.rfc-editor.org/rfc/rfc8446) implementations to efficiently authenticate to peers with diverse [certificate](https://www.rfc-editor.org/rfc/rfc5280) policies.** This avoids a conflict between service availability and user security.
+
+We refer to the authenticating party as the *subscriber* and the peer as the *relying party*. The common case in TLS is server certificate authentication, where the subscriber is the server, and the relying party is the client. The roles are reversed with client certificates. For clarity, this document will primarily discuss the server certificate case, but most of the motivations and solutions apply analogously to client certificates.
+
+To solve this problem, the two drafts extend the existing TLS 1.3 [`certificate_authorities` extension](https://www.rfc-editor.org/rfc/rfc8446#section-4.2.4) to reduce the size cost, making it usable for more applications. They also define a supporting ACME extension to help servers provision multiple certificates. As with other internet drafts at this stage of the process, they are not final mechanisms. Rather, they are starting points, intended to demonstrate feasibility and some approaches, for the working group to build on for a complete solution.
```

Comment:
Suggest: "extend the pattern/mechanism used by the existing TLS 1.3 `certificate_authorities` extension"

Reply:
Done.